Andrespdx/38

andrespdx opened the pull request:
Users are now able to modify the tasks for each project when creating it. Other styles were generally small but I did make few improvements. Let me know if I should add anything else.

andrespdx committed:
added clickable tags to the post a project page

andrespdx committed:
Merge branch 'main' of github.com:daniiltsioma/hobby-dev-hub into andrespdx/38

andrespdx committed:
Added tasks and additional styling

andrespdx committed:
Merge branch 'main' of github.com:daniiltsioma/hobby-dev-hub into andrespdx/38

lbrescia commented:
I like the styling, and it functions well! The only suggestion is to maybe have a way for users to enter multiple tasks when creating the project, but other than that, it is a solid implementation! I do notice that there is a log that prints out when running. I'll attach a photo, but it is from the GitHub API request raw data; I wanted to mention it. 
<img width="583" alt="Screenshot 2025-03-08 at 7 57 01 PM" src="https://github.com/user-attachments/assets/6609939e-4328-4603-9d0e-6d5250a0c230" />

andrespdx committed:
Added support for multiple tags

andrespdx commented:
1. Deleted the console log that was outputting the applicant
2. Added multiple tag support when creating a project
3. Modified the Project page to allow for multiple long tasks
<img width="1067" alt="Screenshot 2025-03-09 at 3 53 09 PM" src="https://github.com/user-attachments/assets/de57a43d-9adb-46ae-a237-eca5651b5554" />

lbrescia commented:
Looks great! Nice, job.

lbrescia commented:
Looks great! Nice job.

lbrescia reviewed (approved):
Looks great!
